.travis.yml

```diff
@@ -2,6 +2,7 @@ language: ruby
 rvm:
   - 2.6.6
   - 2.7.2
+  - 3.0
```

Comment:
Why is there no patch version for 3.0 only?

Reply:
Thanks for your comment. I meant it to specify `3.0.0` .

Reply:
Ruby 3.0.1 has been released and available on Travis CI.
https://rubies.travis-ci.org/

And I'm not sure the exact behavior when omitting the patch version. But maybe not the intended 3.0.0.
https://github.com/rsim/oracle-enhanced/pull/1270

Reply:
Anyway, that's not a big deal. Please merge it with the maintenance policy you like :-)

Reply:
Thanks for the advice.
As far as I can see this line, Travis has used the latest TEENY version of Ruby ( `3.0.1` ) ... 
https://travis-ci.org/github/esminc/seory/jobs/767528000#L316

I'll show the TEENY version of Ruby explicitly that Seory has been tested since it may be helpful for users.